docs/swagger.yml

```diff
@@ -1209,27 +1207,47 @@ definitions:
         type: "string"
         description: "Deletion error message"
         example: "Failed to delete"
-  FactCollectionIn:
+  DeploymentReport:
+    properties:
+        report_id:
+          type: "integer"
+          format: "int64"
+          description: "The id associate with report"
+          example: 1
+        status:
+          type: "string"
+          enum:
+            - pending
+            - complete
+            - failed
+          description: "The status of creating report"
+          example: complete
+        system_fingerprints:
+          type: "array"
+          items:
+            $ref: "#/definitions/SystemFingerprint"
+          description: "The report for the systems that were scanned"
+  DetailsReport:
     type: "object"
     required:
+      - report_id
       - sources
     properties:
+        report_id:
+          type: "integer"
+          format: "int64"
+          description: "The fact collection identifier"
+          example: 15
         sources:
           type: "array"
           items:
             $ref: "#/definitions/FactCollectionSource"
           minItems: 1
-  FactCollectionOut:
+  FactCollectionIn:
```

Comment:
Not a blocker but we should consider making this consistent. ie. FactCollection & Details seem to be the same thing

Reply:
Done